packages/dds/matrix/src/matrix.ts

```diff
@@ -432,7 +504,18 @@ export class SharedMatrix<T = any>
 		);
 		builder.addBlob(
 			SnapshotPath.cells,
-			serializer.stringify([this.cells.snapshot(), this.pending.snapshot()], this.handle),
+			serializer.stringify(
+				[
+					this.cells.snapshot(),
+					this.pending.snapshot(),
+					this.cellLastWriteTracker.snapshot(),
+				],
+				this.handle,
+			),
+		);
+		builder.addBlob(
+			SnapshotPath.setCellConflictResolutionPolicyIsFWW,
```

Comment:
Could you please help me understand why we need this extra blob? I see you added this.cellLastWriteTracker into existing blob. Can we do the same for this info?

Reply:
cellLastWriteTracker was the matrix similar to pending matrix. That is why I added in same blob. We can put this info too in that blob, but are you suggesting this to reduce overhead?

Reply:
Anyway added in the same blob.